spec/extensions/scrabble.spec.js

```diff
@@ -53,5 +53,9 @@ describe("Scrabble", () => {
     it('returns 0 for incorrect tokens', () => {
       expect(scrabble('|d|og')).toEqual(0)
     })
```

Comment:
Like it.
What if I start the word from a closing bracket? like }d}og
Will the program still run?

Reply:
It will! at the very start I check for "the word contains only valid characters" and then go from there. This is where d|o|g gets thrown out for the pipe characters. it'll also throw out }d}og for having an uneven number of braket pairs / trying to lower the factor below 1 (you start at 1, and } tries to get it to 1/2, which sets charFactor.valid to false (line 43)